### PR TITLE
Cypress testing

### PR DESCRIPTION
**Pull request by BriBourassa**
Summary: 
- Rework entire story in story-data.js
- Add Cypress testing

### CODE REVIEW

**Review by Jesusborjas006 — APPROVED** [review]
Great job testing!!

File: src/data/story-data.js
Comment (qrispi) [inline src/data/story-data.js:1]: Thanks for fixing up the story line!

File: cypress/fixtures/cat-fact.json
Comment (qrispi) [inline cypress/fixtures/cat-fact.json:1]: Those darn fixtures!

**Review by qrispi — APPROVED** [review]
Awesome work! Thanks for spear-heading the testing!

**Pull request merged**